### PR TITLE
cephfs: expand clone error messages

### PR DESCRIPTION
**Pull request by agarwal-mudit**
Adding "snapshot clone" in the clone error messages.

Signed-off-by: Mudit Agarwal <muagarwa@redhat.com>

### CODE REVIEW

**Comment by agarwal-mudit** [comment]
Just a nitpick as I observed following message while debugging a downstream issue:

> I0422 08:31:54.988126       1 controller.go:642] CreateVolume failed, supports topology = false, node selected false => may reschedule = false    => state = Background: rpc error: code = Aborted desc = in progress
> I0422 08:32:32.633750       1 controller.go:1091] Temporary error received, adding PVC 4b61e5ee-0ffc-41d5-b2a8-fa6ee5fd6ed4 to claims in progress
> W0422 08:32:32.633771       1 controller.go:943] Retrying syncing claim "4b61e5ee-0ffc-41d5-b2a8-fa6ee5fd6ed4", failure 6
> E0422 08:32:32.633797       1 controller.go:966] error syncing claim "4b61e5ee-0ffc-41d5-b2a8-fa6ee5fd6ed4": failed to provision volume with      StorageClass "ocs-storagecluster-cephfs": rpc error: code = Aborted desc = in progress
> 

In case of a volume restore operation, if volume creation fails because the clone is in progress or pending it just says "pending" or "in progress". No one can tell by reading the message that clone failed or clone in progress.

**Review by Yuggupta27 — APPROVED** [review]

File: internal/cephfs/errors.go [hunk 1]
```diff
@@ -28,10 +28,10 @@ const (
 
 var (
 	// ErrCloneInProgress is returned when snapshot clone state is `in progress`
-	ErrCloneInProgress = errors.New("in progress")
+	ErrCloneInProgress = errors.New("snapshot clone in progress")
 
 	// ErrClonePending is returned when snapshot clone state is `pending`
-	ErrClonePending = errors.New("pending")
+	ErrClonePending = errors.New("snapshot clone pending")
```
Comment (Yuggupta27) [inline internal/cephfs/errors.go]:
```suggestion
	ErrClonePending = errors.New("snapshot clone is pending")
```

**Review by Yuggupta27 — CHANGES_REQUESTED** [review]

File: internal/cephfs/errors.go (hunk 1 above)
Comment (Yuggupta27) [reply internal/cephfs/errors.go]: A minor nit

File: internal/cephfs/errors.go [hunk 2]
```diff
@@ -28,10 +28,10 @@ const (
 
 var (
 	// ErrCloneInProgress is returned when snapshot clone state is `in progress`
-	ErrCloneInProgress = errors.New("in progress")
+	ErrCloneInProgress = errors.New("snapshot clone in progress")
```
Comment (humblec) [inline internal/cephfs/errors.go]: May be "clone from snapshot is in progress already" make it better I believe

**Comment by humblec** [comment]
Thanks @agarwal-mudit . A minor suggestion from my end.

File: internal/cephfs/errors.go (hunk 1 above)
Comment (agarwal-mudit) [reply internal/cephfs/errors.go]: fixed

**Review by humblec — APPROVED** [review]

**Review by Yuggupta27 — APPROVED** [review]

**Comment by Yuggupta27** [comment]
> /retest ci/centos/mini-e2e/k8s-1.19

deploy-rook failed. 
```
Error from server: error when creating "/tmp/tmp.qiS0NSRkHG/cluster-test.yaml": etcdserver: request timed out
```

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**